### PR TITLE
Save the change set into ChangeLog record.

### PR DESCRIPTION
**Pull request by vcraescu**
Added a new options: `loggable.ComputeDiff`
When this options is set, all changed fields tagged with `gorm-loggable: true` will be saved into `ChangeLog.RawDiff`. Also, CreatedBy field was added to the record because that's quite common to know who did the change and when. Most of the time you will want to just display a history of changes and who did those changes and when.

Tried to replicate Doctrine Loggable Extension behaviour.

### CODE REVIEW

**Comment by vcraescu** [comment]
ping!

**Comment by vcraescu** [comment]
ping!

File: loggable.go [hunk 1]
```diff
@@ -42,6 +42,8 @@ type ChangeLog struct {
 	ObjectType string      `gorm:"index"`
 	RawObject  string      `sql:"type:JSON"`
 	RawMeta    string      `sql:"type:JSON"`
+	RawDiff    string      `sql:"type:JSONB"`
```
Comment (sas1024) [inline loggable.go]: It's better to use ANSI JSON, because jsonb it's only postgresql feature.

**Comment by sas1024** [comment]
> ping!

pong :)

**Comment by sas1024** [comment]
One more thing - it's need to describe new feature in README.md

File: loggable.go (hunk 1 above)
Comment (vcraescu) [reply loggable.go]: Sure. Just missed that during the merge. :)

**Comment by vcraescu** [comment]
@sas1024 I did the changes you requested. Please check!

Thanks!

**Comment by vcraescu** [comment]
ping! :smile:

**Pull request merged**